platforms/android/library/src/androidTest/java/io/element/android/wysiwyg/test/utils/ImeActions.kt

```diff
@@ -126,9 +125,7 @@ object Ime {
             if (view == null) return
             val editorInfo = EditorInfo()
             val inputConnection = view.onCreateInputConnection(editorInfo) as? InterceptInputConnection ?: return
-            val end = inputConnection.getCurrentComposition().second
-            inputConnection.setComposingRegion(end, end)
-            inputConnection.commitText("\n", 1)
+            inputConnection.onHardwareEnterKey()
```

Comment:
`ImeAction`s were implemented as the software keyboard counterpart of the `pressKey` actions that use the hardware keyboard input, so calling a method  `onHardware___Key` here feels a bit weird, but since the implementations should match now it should be fine.